EvolutionModelTests/WateringHoleTests.cs

```diff
@@ -54,5 +54,29 @@ public class WateringHoleTests {
 
       Assert.AreEqual(0, wateringhole.FoodAmount);
     }
+
+        [TestMethod]
+        public void WaterHoleTryEatingCanEat()
+        {
+
+        }
+
+        [TestMethod]
+        public void WaterHoleEatingCanEat()
```

Comment:
Wat betekent Can Eat? Waar test je voor? Wat is je input? Wat is de verwachte output/side-effecits?